packages/Laravel/src/Rector/FuncCall/ArrayHelperToIlluminateSupportClassRector.php

```diff
@@ -0,0 +1,162 @@
+        array_sort_recursive($data);
+        array_where($data, static function ($value) { return $value > 5;});
+        array_wrap($data);
+    }
+}
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+class SomeClass
+{
+    public function run($data)
+    {
+        \Illuminate\Support\Arr::add($data, 'field', 'value');
+        \Illuminate\Support\Arr::collapse($data);
+        \Illuminate\Support\Arr::divide($data);
+        \Illuminate\Support\Arr::dot($data);
+        \Illuminate\Support\Arr::dot($data, '-');
+        \Illuminate\Support\Arr::except($data, 'ignore');
+        \Illuminate\Support\Arr::first($data);
+        \Illuminate\Support\Arr::first($data, static function ($value) {$value > 5;});
+        \Illuminate\Support\Arr::first($data, static function ($value) {$value > 5;}, 10);
+        \Illuminate\Support\Arr::flatten($data);
+        \Illuminate\Support\Arr::flatten($data, 5);
+        \Illuminate\Support\Arr::forget($data, 'forget_this');
+        \Illuminate\Support\Arr::get($data, 'field');
+        \Illuminate\Support\Arr::get($data, 'field', 'default');
+        \Illuminate\Support\Arr::has($data, 'needed');
+        \Illuminate\Support\Arr::last($data);
+        \Illuminate\Support\Arr::last($data, static function ($value) {$value > 5;});
+        \Illuminate\Support\Arr::last($data, static function ($value) {$value > 5;}, 10);
+        \Illuminate\Support\Arr::only($data, 'key');
+        \Illuminate\Support\Arr::pluck($data, 'value');
+        \Illuminate\Support\Arr::pluck($data, 'value', 'key');
+        \Illuminate\Support\Arr::prepend($data, 'value');
+        \Illuminate\Support\Arr::prepend($data, 'value', 'key');
+        \Illuminate\Support\Arr::pull($data, 'value');
+        \Illuminate\Support\Arr::pull($data, 'value', 'key');
+        \Illuminate\Support\Arr::random($data);
+        \Illuminate\Support\Arr::random($data, 5);
+        \Illuminate\Support\Arr::set($data, 'key', 'value');
+        \Illuminate\Support\Arr::sort($data);
+        \Illuminate\Support\Arr::sort($data, static function ($value) {return strrev($value);});
+        \Illuminate\Support\Arr::sortRecursive($data);
+        \Illuminate\Support\Arr::where($data, static function ($value) { return $value > 5;});
+        \Illuminate\Support\Arr::wrap($data);
+    }
+}
+CODE_SAMPLE
+            ),
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    public function getNodeTypes(): array
+    {
+        return [FuncCall::class];
+    }
+
+    /**
+     * @param FuncCall $node
+     */
+    public function refactor(Node $node): ?Node
+    {
+        foreach ($this->helpersToFunction as $helperFunction => $classFunction) {
+            if (! $this->isName($node, $helperFunction)) {
+                continue;
+            }
+
+            return new StaticCall(new Name($this->supportClass), new Identifier($classFunction), $node->args);
+        }
+        return $node;
```

Comment:
If the node hasn't been modified, `null` should be returned (screen from README)
![image](https://user-images.githubusercontent.com/23243361/63011960-016e0180-be92-11e9-824f-a1a735795382.png)

Reply:
Thanks for the feedback, I have changed the function to return null.